pkg/systemd/service-details.jsx

```diff
@@ -421,100 +421,154 @@
-                    <ErrorCircleOIcon className="status-icon" />
+            statuses.push(
+                <ListItem
+                    key="failed"
+                    className="status-failed"
+                    icon={
+                        <Icon>
+                            <ErrorCircleOIcon className="status-icon" />
+                        </Icon>
+                    }
+                >
                     <span className="status">{ _("Failed to start") }</span>
                     { showAction &&
-                    <Button variant="secondary" className="action-button" onClick={() => this.unitAction("StartUnit") }>{ _("Start service") }</Button>
+                        <Button variant="secondary" className="action-button" onClick={() => this.unitAction("StartUnit")}>
+                            {_("Start service")}
+                        </Button>
                     }
-                </div>
+                </ListItem>
             );
         }
 
-        if (!status.length) {
+        if (!statuses.length) {
             if (active) {
-                status.push(
-                    <div key="running" className="status-running">
-                        <Icon status="success">
-                            <OnRunningIcon className="status-icon" />
-                        </Icon>
-                        <span className="status">{ _("Running") }</span>
-                        <span className="side-note font-xs">{ _("Active since ") + timeformat.dateTime(this.props.unit.ActiveEnterTimestamp / 1000) }</span>
-                    </div>
+                statuses.push(
+                    <ListItem
+                        key="running"
+                        className="status-running"
+                        icon={
+                            <Icon>
+                                <OnRunningIcon className="status-icon" />
+                            </Icon>
+                        }
+                    >
+                        <span className="status">{_("Running")}</span>
+                        <span className="side-note font-xs">{_("Active since ") + timeformat.dateTime(this.props.unit.ActiveEnterTimestamp / 1000)}</span>
+                    </ListItem>
                 );
             } else {
-                status.push(
-                    <div key="stopped" className="status-stopped">
-                        <Icon>
-                            <OffIcon className="status-icon" />
-                        </Icon>
-                        <span className="status">{ _("Not running") }</span>
-                    </div>
+                statuses.push(
+                    <ListItem
+                        key="stopped"
+                        className="status-stopped"
+                        icon={
+                            <Icon>
+                                <OffIcon className="status-icon" />
+                            </Icon>
```

Comment:
These 8 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22251-16fefbfb-20250725-220859-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)